composer.json

```diff
@@ -80,10 +80,10 @@
 			"composer du"
 		],
 		"prefix-ruckusing": [
-			"./vendor/bin/php-scoper add-prefix --prefix=YoastSEO_Vendor --output-dir=./vendor_prefixed/ruckusing --config=config/php-scoper/ruckusing.inc.php --force --quiet"
+			"@php ./vendor/humbug/php-scoper/bin/php-scoper add-prefix --prefix=YoastSEO_Vendor --output-dir=../../../vendor_prefixed/ruckusing --config=config/php-scoper/ruckusing.inc.php --force"
```

Comment:
Please restore the `--quite` in this command :)

Reply:
Thanks for catching that. I've fixed up the commit.